201-windows-vm-diagnostics-extension/README.md

```diff
@@ -3,5 +3,8 @@
 <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-premium-storage-windows-vm-diagnostics-extension%2Fazuredeploy.json" target="_blank">
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
+<a href="http://armviz.io/#/?load=https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-windows-vm-diagnostics-extension/azuredeploy.json" target="_blank">
```

Comment:
Shows a storage account that is not connected to the VM. Can you check what's going on?